<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataSupir</class>
 <widget class="QDialog" name="DataSupir">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/res/WhatsApp Image 2024-05-07 at 19.42.08_2aaf5d92.jpg);</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
border-image: url(:/res/white.jpg);
image: url(:/res/white.jpg);
</string>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="idLineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/res/orange.jpg);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="namaLineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/res/orange.jpg);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
image: url(:/res/white.jpg);
border-image: url(:/res/white.jpg);
</string>
   </property>
   <property name="text">
    <string>Nama</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nikLineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/res/orange.jpg);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
color: rgb(0, 0, 0);
image: url(:/res/white.jpg);
border-image: url(:/res/white.jpg);</string>
   </property>
   <property name="text">
    <string>NIK</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="alamatLineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>190</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/res/orange.jpg);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
image: url(:/res/white.jpg);
border-image: url(:/res/white.jpg);</string>
   </property>
   <property name="text">
    <string>Alamat</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="jaminanLineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>230</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/res/orange.jpg);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>230</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
image: url(:/res/white.jpg);
border-image: url(:/res/white.jpg);</string>
   </property>
   <property name="text">
    <string>Jaminan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>300</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/res/orange.jpg);
image: url(:/res/orange.jpg);</string>
   </property>
   <property name="text">
    <string>Simpan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="updateButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>300</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/res/orange.jpg);
image: url(:/res/orange.jpg);</string>
   </property>
   <property name="text">
    <string>Ubah</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>300</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/res/orange.jpg);
image: url(:/res/orange.jpg);</string>
   </property>
   <property name="text">
    <string>Hapus</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>531</width>
     <height>192</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/res/orange.jpg);
border-color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sortButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>300</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/res/orange.jpg);
image: url(:/res/orange.jpg);</string>
   </property>
   <property name="text">
    <string>Urutkan</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/res/white.jpg);
image: url(:/res/white.jpg);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Data Supir</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>label_2</zorder>
  <zorder>namaLineEdit</zorder>
  <zorder>label_3</zorder>
  <zorder>nikLineEdit</zorder>
  <zorder>label_4</zorder>
  <zorder>alamatLineEdit</zorder>
  <zorder>label_5</zorder>
  <zorder>jaminanLineEdit</zorder>
  <zorder>label_6</zorder>
  <zorder>addButton</zorder>
  <zorder>updateButton</zorder>
  <zorder>deleteButton</zorder>
  <zorder>tableView</zorder>
  <zorder>sortButton</zorder>
  <zorder>idLineEdit</zorder>
  <zorder>label_7</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
